docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+  intended audience (`aud`) claim of the submitted access token. Also required
+  when API authentication is enabled.
+* `skipEmailVerifiedCheck`: One of the standard claim elements that can be
+  requested and/or returned in an OIDC ID token or UserInfo response is the
+  `email_verified` claim - When this claim value is true, this means that the
+  OIDC provider "took affirmative steps to ensure that this e-mail address was
+  controlled by the End-User at the time the verification was performed." When
+  `skipEmailVerifiedCheck` is set to **false**, Enduro will check any submitted
+  tokens or UserInfo responses for the verified email claim. When set to
+  **true**, this check is skipped.
+
+#### Enable Attribute Based Access Control for the API OIDC authentication
+
+Enduro uses Attribute Based Access Control ([ABAC]) to manage permissions and
+access. When ABAC is enabled for the API, it will check a configurable
+multivalue claim against the defined required attributes based on each
+endpoint's configuration.
+
+**Default values**:
+
+```toml
+[api.auth.oidc.abac]
+enabled = true
+claimPath = "enduro"
+claimPathSeparator = ""
+claimValuePrefix = ""
+useRoles = false
+rolesMapping =
+```
+
+* `enabled`: Accepts `true` or `false` as values.
+* `claimPath`: The claim path of the Enduro attributes within the access token.
+  If the claim path is nested, include all fields separated by the
+  `claimPathSeparator` defined below. This element is required when `enabled` is
+  set to "true."
+* `claimPathSeparator`: The separator used to split nested claim path fields.
+  The default empty value of "" will try to match the claim path as-is to a
+  top-level field from the access token.
+* `claimValuePrefix`: Used to add a prefix to filter the values of the
+  configured claim. If the claim contains values unrelated to Enduro's ABAC, the
+  values relevant to Enduro should be prefixed so they are the only values used
+  for access control.
+
+    For example, a claim with values `["enduro:*", "unrelated"]` will be
+    filtered to a value of `["*"]` when `claimValuePrefix = "enduro:"`.
+
+    The default empty value of "" will not filter any value.
+
+* `useRoles`: To simplify access control management for smaller organizations or
+  those not requiring fine-grained access control, it is possible to predefine
+  a set of roles with specific attributes, to which users can be assigned. Set
+  this value to `true` to switch to role-based access control, and define the
+  attributes associated with each role in the `rolesMapping` element below. This
+  value is set to `false` by default.
+* `rolesMapping`: If `useRoles` is set to true, this element is where you can
+  specify a JSON-formatted string mapping each role you'd like to define to a
+  set of attributes. The expected format is as follows:
+
+    ```json
+    {
+      "role1": ["attribute1", "atrribute2"],
+      "role2": ["attribute1", "atrribute2", "attribute3", "atrribute4"]
```

Comment:
Typo: "atrribute2"

Reply:
oops, another error copied directly from the TOML file. Fixed there as well.